modules/core/server/routes/core.server.routes.js

```diff
@@ -22,6 +22,9 @@ module.exports = function(app) {
   // Return a 404 for all undefined api, module or lib routes
   app.route('/:url(api|modules|lib|developers)/*').get(core.renderNotFound);
 
+  // Gives the service worker access to any config it needs
+  app.route('/config/sw.js').get(core.renderServiceWorkerConfig);
```

Comment:
Normally these been exposed to index html template from [config/lib/express.js](https://github.com/Trustroots/trustroots/blob/master/config/lib/express.js#L31).